src/Visitors/RemoveArrayPropertyItem.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace RonasIT\Larabuilder\Visitors;
+
+use Illuminate\Support\Arr;
+use PhpParser\ConstExprEvaluator;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Scalar;
+use PhpParser\Node\Stmt\Property;
+use RonasIT\Larabuilder\Exceptions\UnexpectedPropertyTypeException;
+
+class RemoveArrayPropertyItem extends PropertyValueAbstract
+{
+    public function __construct(
+        string $name,
+        protected array $valuesToRemove,
+    ) {
+        parent::__construct($name);
+    }
+
+    /** @param Property $node */
+    protected function updateNode(Node $node): void
+    {
+        $arrayProperty = $node->props[0]->default;
+
+        if (!$arrayProperty instanceof Array_) {
+            throw new UnexpectedPropertyTypeException(
+                property: $this->name,
+                expectedType: 'array',
+                actualType: (string) $node->type,
```

Comment:
the `null` stringification will return '', so the exception will never receive `null` as arg. It would be better to return the real null and convert it to the 'null' in the exception OR convert it here and remove nullable in exception constructor declaration